Makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: all binary ut clean
+.PHONY: all binary ut clean dist/calico_kubernetes
```

Comment:
A better way of doing this is probably to just replace all instances of `dist/calico_kubernetes` with `binary`.